plugins/PostProcessingPlugin/scripts/Stretch.py

```diff
@@ -83,6 +86,24 @@ def copyPosFrom(self, step):
     def setInRelativeMovement(self, value: bool) -> None:
         self.in_relative_movement = value
 
+class GCodeStepSlot:
+    """
+    GCodeStep was originally used both during input parsing and during processing and output writing. In order to
+    work with multiprocessing pools, the step objects need to be manually pickled (classes in plugins aren't
+    directly pickleable). Rather than reconstruct GCodeSteps again, this class is used during processing. The hope
+    is that, by decoupling the two, future improvements will be easier.
+    """
```

Comment:
Please put the documentation in Doxygen-style comments above the class, for now.